src/helpers/indexable-helper.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * A helper object for indexables.
+ *
+ * @package Yoast\WP\SEO\Helpers
+ */
+
+namespace Yoast\WP\SEO\Helpers;
+
+use Yoast\WP\SEO\Models\Indexable;
+
+/**
+ * Class Indexable_Helper
+ */
+class Indexable_Helper {
+	/**
+	 * Retrieves the permalink for an indexable.
+	 *
+	 * @param Indexable $indexable The indexable.
+	 *
+	 * @return string|null The permalink.
+	 */
+	public function get_permalink_for_indexable( $indexable ) {
+		switch ( true ) {
+			case $indexable->object_type === 'post':
```

Comment:
Nothing happens in this case?

Reply:
It either this case or the one below. You can read this as:
```
if ( object_type === 'post' || object_type === 'home-page' ) {
```